BUILD

```diff
@@ -2275,6 +2275,103 @@ grpc_cc_library(
 )
 
 # TODO: Get this into build.yaml once we start using it.
+grpc_cc_library(
+    name = "xds_api",
```

Comment:
Suggest prefixing each of these target names with "upb_", so that it's clear what they contain.